openapi-contracts/src/test/resources/domains/numbers/v1/active/active-numbers-update.json

```diff
@@ -0,0 +1,12 @@
+{
+  "displayName": "a display",
+  "smsConfiguration": {
+    "servicePlanId": "service plan id",
+    "campaignId": "campaign id"
+  },
+  "voiceConfiguration": {
```

Comment:
This payload is used in a request context, so these fields do not have sense here
Renamed the file to highlight a "request" payload
Response payload is covered by the "GET" test because update is returning an AcitveNumber payload like a GET